packages/basic/src/utils/json-bigint/lib/stringify.js

```
@@ -0,0 +1,379 @@
+    switch (typeof value) {
+      case 'string':
+        if (isBigNumber) {
+          return value;
+        } else {
+          return quote(value);
+        }
+
+      case 'number':
+        // JSON numbers must be finite. Encode non-finite numbers as null.
+
+        return isFinite(value) ? String(value) : 'null';
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)